core/src/main/java/com/yahoo/ycsb/GoodBadUglyDB.java

```diff
@@ -1,162 +1,159 @@
- * Unless required by applicable law or agreed to in writing, software                                                                                                             
- * distributed under the License is distributed on an "AS IS" BASIS,                                                                                                               
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or                                                                                                                 
- * implied. See the License for the specific language governing                                                                                                                    
- * permissions and limitations under the License. See accompanying                                                                                                                 
- * LICENSE file.                                                                                                                                                                   
+/**
+ * Copyright (c) 2010 Yahoo! Inc. All rights reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
  */
 
 package com.yahoo.ycsb;
 
-import static java.util.concurrent.TimeUnit.MICROSECONDS;
-
 import java.util.HashMap;
 import java.util.Random;
 import java.util.Set;
 import java.util.Vector;
-import java.util.concurrent.TimeUnit;
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.LockSupport;
 import java.util.concurrent.locks.ReadWriteLock;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 
+import static java.util.concurrent.TimeUnit.MICROSECONDS;
+
 /**
  * Basic DB that just prints out the requested operations, instead of doing them against a database.
  */
 public class GoodBadUglyDB extends DB {
-    public static final String SIMULATE_DELAY = "gbudb.delays";
-    public static final String SIMULATE_DELAY_DEFAULT = "200,1000,10000,50000,100000";
-    long delays[];
-    static ReadWriteLock DB_ACCESS = new ReentrantReadWriteLock();
+  public static final String SIMULATE_DELAY = "gbudb.delays";
+  public static final String SIMULATE_DELAY_DEFAULT = "200,1000,10000,50000,100000";
+  private long[] delays;
+  private static final ReadWriteLock DB_ACCESS = new ReentrantReadWriteLock();
```

Comment:
Nother nit regarding the privatization of these two. Shouldn't be anyone using them but may be good to set them to ``protected`` just to be safe.